package.json

```diff
@@ -8,8 +8,7 @@
     "lint": "eslint 'app/**/*.js'",
     "server": "node app/server.js",
     "test": "jest --watch",
-    "test:coverage": "jest --coverage",
-    "precommit": "lint-staged"
```

Comment:
Latest version of husky moves `precommit` script out from here and into `husky.hooks.pre-commit`